CoreFoundation/Locale.subproj/CFLocale.c

```diff
@@ -1429,20 +1478,16 @@ static bool __CFLocaleCopyCollatorID(CFLocaleRef locale, bool user, CFTypeRef *c
 }
 
 #if DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED
-static CONST_STRING_DECL(_metricUnitsKey, "AppleMetricUnits");
-static CONST_STRING_DECL(_measurementUnitsKey, "AppleMeasurementUnits");
-static CONST_STRING_DECL(_measurementUnitsCentimeters, "Centimeters");
-static CONST_STRING_DECL(_measurementUnitsInches, "Inches");
-static CONST_STRING_DECL(_temperatureUnitKey, "AppleTemperatureUnit");
+STATIC_CONST_STRING_DECL(_metricUnitsKey, "AppleMetricUnits");
+STATIC_CONST_STRING_DECL(_measurementUnitsKey, "AppleMeasurementUnits");
+STATIC_CONST_STRING_DECL(_measurementUnitsCentimeters, "Centimeters");
+STATIC_CONST_STRING_DECL(_measurementUnitsInches, "Inches");
+STATIC_CONST_STRING_DECL(_temperatureUnitKey, "AppleTemperatureUnit");
 
 static bool __CFLocaleGetMeasurementSystemForPreferences(CFTypeRef metricPref, CFTypeRef measurementPref, UMeasurementSystem *outMeasurementSystem) {
     if (metricPref || measurementPref) {
         if (metricPref == kCFBooleanTrue && measurementPref && CFEqual(measurementPref, _measurementUnitsInches)) {
-#if U_ICU_VERSION_MAJOR_NUM >= 55
```

Comment:
Will removing this check break Ubuntu14 builds which I think only has version 52?

Reply:
Good catch.